docs/reference/extensions.rst

```diff
@@ -1,137 +1,134 @@
-Extensions
```

Comment:
Hey @VincentLanglet could I rename this file to `customize_admin.rst`?

Reply:
it's missing only this question be answered 👀

Reply:
we don't have too much benefits with it, it's more to keep the title equals file name!

But if I do this and others docs links with this page, it'll return not found!